src/pages/jobs.md

```diff
@@ -185,3 +192,147 @@ If you're interested in joining us, please email eric@shorebird.dev. You can
+
+Your job is to help us reach those .9M developers. We’re so far 3 engineers and
+an operations lead. None of us love marketing like you do. You will help us
+craft our story and tell it widely. You love talking to developers, you love
+writing content, giving talks etc and you’ll have full support to do what it
+takes to raise awareness. At a small company this is a generalist role. You
+don’t have to be amazing at all parts of developer relations or marketing, but
+no one else is here to do things if one of us doesn’t.
+
+### What You’ll Do
+
+- Grow developer awareness. Craft and execute strategies to get Shorebird in
+  front of more Flutter developers.
+
+- Grow our social presence. Own and operate Shorebird’s social presence across
+  all channels.
+
+- Write and speak. Publish technical blog posts, create videos or live demos,
+  speak at conferences, and engage with the community on X, GitHub, Reddit, and
+  beyond.
+
+- Advocate for Flutter and Shorebird. Shorebird only succeeds if Flutter does.
+  You get to tell both stories to customers at all phases of their adoption
+  journey.
+
+- Build relationships. Identify and support influential Flutter developers and
+  early adopters.
+
+- Close the loop. Bring insights from the community back to the team to inform
+  product direction.
+
+- Own marketing. From website content to email campaigns to positioning, you’ll
+  shape how we tell the Shorebird story.
+
+### What We’re Looking For
+
+- Mission alignment. You care about helping the world stop writing everything
+  twice. Flutter is our tool for that and Shorebird aims to take Flutter beyond
+  where Google can.
+
+- Strong desire to work at a startup. We’re a tiny team. All remote. You’ll be
+  the first person fulltime driving awareness and will get to define our
+  strategy and approach.
+
+- Scrappy and self-driven. You are the marketing department. You are our DevRel
+  org. You’ll have lots of help, but you’ll have to want to make your own
+  decisions and own them. You’ll also (likely) be fully remote which means you
+  need to be great at self motivation and self-direction.
+
+- Experience working in Developer Relations, Product Marketing, or Technical
+  Advocacy — or strong proof you can do the job.
+
+- Strong writing and communication skills. You enjoy breaking down complex
+  topics for different audiences.
+
+- Comfort with code. You don’t need to be an expert Flutter dev, but you can
+  read docs, run sample apps, and maybe write a working blog demo.
+
+- Bonus: experience in startups, open-source communities, or the Flutter
+  ecosystem.
+
+### Why Join
+
+- Work directly with a small, senior team solving hard, high-impact problems.
+
+- Influence product, messaging, and strategy from the ground up.
+
+- Competitive compensation \+ early equity.
+
+- Shape the future of how mobile apps are built and shipped.
+
+## Generalist Software Engineer
+
+Location: Remote or Chicago / Brooklyn / Palo Alto
```

Comment:
```suggestion
Location: Remote, candidates based in Chicago, Brooklyn, or Palo Alto are especially encouraged to apply.
```

Reply:
Or combine this with the location block you have at the bottom. Doesn't make sense to repeat